data/dashboard.yml

```diff
@@ -57,8 +57,8 @@
         - govuk-cdn-config
         - router
       sites:
```

Comment:
You can also remove this section and add `router-data` to the `repos:` array above, so that the repo description is auto-updated.